DigitalLearningSolutions.Data.Tests/Services/BrandsServiceTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Models.Common;
+    using DigitalLearningSolutions.Data.Models.Support;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using FakeItEasy;
+    using FizzWare.NBuilder;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using NUnit.Framework;
+
+    public class BrandsServiceTests
+    {
+        private IBrandsDataService brandsDataService = null!;
+        private BrandsService brandsService = null!;
+
+        [SetUp]
+        public void Setup()
+        {
+            brandsDataService = A.Fake<IBrandsDataService>();
+            brandsService = new BrandsService(brandsDataService);
+        }
+
+        [Test]
+        public void GetAllPublicBrands_calls_data_service_method_and_returns_public_results()

```

Comment:
The method name in the test name should match the method being called in the `// When` section